static/main.js

```diff
@@ -406,10 +406,10 @@ const other_aliases = {
     "ø↳": [/c(ustom)?_?pad_?l(eft)?/], // Frick
     "↲": [/shift_?l(eft)?/,/l(eft)?_?pad/,/pad_?l(eft)?/],
     "ø↲": [/c(ustom)?_?pad_?r(ight)?/],
-    "⋏": [/bit(wise)?_?and/,/pad_?c(enter)?|c(enter)?_?pad/],
-    "⋎": [/bit(wise)?_?or/,/(rm|remove)_?ind(ex)?/,"merge"],
-    "꘍": [/bitwise_?xor/,/l(even(shtein)?)?_?dist(ance)?/,/(add|prepend|append)_?spaces?/],
-    "ꜝ": [/bitwise_?not/,/filter_?fals[ye]/,/any_?upper(case)?/],
+    "⋏": [/bit(wi[sz]e)?_?and/,/pad_?c(enter)?|c(enter)?_?pad/],
+    "⋎": [/bit(wi[sz]e)?_?or/,/(rm|remove)_?ind(ex)?/,"merge"],
+    "꘍": [/bitwi[sz]e_?xor/,/l(even(shtein)?)?_?dist(ance)?/,/(add|prepend|append)_?spaces?/],
+    "ꜝ": [/bitwi[sz]e_?not/,/filter_?fals[ye]/,/any_?upper(case)?/],
```

Comment:
What on earth, bitwize is not a word.

Reply:
Oh. I'm bad at American :P

Reply:
Keep it. This is exactly how we Americanz zpeak